src/net/sourceforge/kolmafia/request/coinmaster/AltarOfBonesRequest.java

```diff
@@ -34,12 +34,9 @@ public AltarOfBonesRequest(final boolean buying, final AdventureResult[] attachm
     super(ALTAR_OF_BONES, buying, attachments);
   }
 
-  public AltarOfBonesRequest(final boolean buying, final AdventureResult attachment) {
-    super(ALTAR_OF_BONES, buying, attachment);
-  }
-
-  public AltarOfBonesRequest(final boolean buying, final int itemId, final int quantity) {
-    super(ALTAR_OF_BONES, buying, itemId, quantity);
+  @Override
+  public void processResults() {
```

Comment:
was this a deliberate change as part of this PR? It wasn't called out.

Reply:
It was deliberate: it was a bug in the obsolete class.

ResponseTextParser has
```
      case "bone_altar.php" -> {
        AltarOfBonesRequest.parseResponse(location, responseText);
      }
```
but that is invoked only if this was an "external" request.
If you instantiate and run the class, without a processResults, it would never parse the response, since that is not an external request.

Reply:
I added it to the PR description. Sorry!